### PR TITLE
Adds Armbands to loadout

### PR DESCRIPTION
**Pull request by Aurorablade**
Adds armbands for most departments to the loadout plus a red and blue yellow thats for everyone.

I did take one or two from bay.

:cl: Fethas
rscadd: ARMBANDS! REPRESENT! SEE THE LOADOUT!
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Armbands serve little purpose equipped to the job they are actually part of, but I guess you can argue any accessory at that.
I always saw armbands as a way to show you are part of a department while not wearing their uniform. Like assistants or maybe Botany wanting to Science plants outside of the drug den. Maybe an engineer helps out on the mining station. Something that says "I am this department but I'm also working in this other one too"

**Comment by DesolateG** [comment]
@taukausnake

OR to show off that you're a fascist security force.

**Comment by MarcellusPye** [comment]
Atmos does not have own armband, 0/meme, do not merge

**Comment by Aurorablade** [comment]
i was too lazy to recolor anything at the time.

**Comment by SamHPurp** [comment]
Are the medical ones, Paramed colour or TGMed colour - because the two stations have different colours for their Medical...

File: code/modules/clothing/under/accessories/armband.dm [hunk 1]
```diff
@@ -5,6 +5,18 @@
...
+/obj/item/clothing/accessory/armband/sec
+	name = "security armband"
+	desc = "An armband, worn by the crew to display which department they're assigned to. This one is white and red."
+	icon_state = "whitered"
+	item_color = "whitered"
+
+/obj/item/clothing/accessory/armband/yb
+	name = "blue-yellow armband"
+	desc = "A fancy blue and yellow armband!"
+	icon_state = "whitered"
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/armband.dm]: 🤔

Comment (Aurorablade) [reply code/modules/clothing/under/accessories/armband.dm]: I HAVE NO IDEA WHAT YOU ARE TALKING ABOUT EVERYTHING LOOKS NORMAL

Comment (taukausanake) [reply code/modules/clothing/under/accessories/armband.dm]: Okay, NOW I see it

**Comment by EldritchSigma** [comment]
I read armblades... was like "What?!" Then I saw "Armbands".

**Pull request merged**